airflow/decorators/base.py

```diff
@@ -46,7 +47,7 @@
 from airflow.utils.task_group import TaskGroup, TaskGroupContext
 
 
-def validate_python_callable(python_callable):
+def validate_python_callable(python_callable: Any) -> None:
```

Comment:
I wonder if `Callable` type could be used here?

```suggestion
def validate_python_callable(python_callable: Callable) -> None:
```

Reply:
No because this function accepts non-callables; it validates the input is a callable, and throws an exception when it’s not. So we _don’t_ want Mypy to complain when we do (say) `validate_python_callable(1)`; it’s an expected usage.